Classes/Controller/Backend/FormDataController.php

```diff
@@ -134,7 +136,7 @@ public function previewAction(FormData $formDataEntry): void
      */
     public function deleteAction(FormData $formDataEntry): void
     {
-        $this->formDataRepository->removeByFormIdentifierAndHash($formDataEntry->getFormIdentifier(), $formDataEntry->getHash());
+        $this->formDataRepository->removeByFormIdentifierAndHash($formDataEntry->getFormIdentifier(), $formDataEntry->getHash(), $formDataEntry->getSitename(), $formDataEntry->getDimensionsHash());
```

Comment:
Should also be named removeByFormProperties

Reply:
renamed ✔️